### PR TITLE
Fix reading no headers

### PR DESCRIPTION
**Pull request by levik-buka**
May be it is a bug in RabbitMQ Client library, but when publishing messages without headers in RabbitMQ Management UI,
basicProperties.IsHeadersPresent() returns true, but basicProperties.Headers is null.

Also added check if header is null or non byte[] type.

### CODE REVIEW

File: Frends.Community.RabbitMQ.Tests/UnitTests.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class UnitTests
         public void DeleteExchangeAndQueue()
         {
             var factory = new ConnectionFactory
-            {
+        {
```
Comment (RikuVirtanen) [inline Frends.Community.RabbitMQ.Tests/UnitTests.cs:37]: Incorrect indentation.

File: Frends.Community.RabbitMQ.Tests/UnitTests.cs [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void DeleteExchangeAndQueue()
         public void CreateExchangeAndQueue()
         {
             var factory = new ConnectionFactory
-            {
+        {
```
Comment (RikuVirtanen) [inline Frends.Community.RabbitMQ.Tests/UnitTests.cs:58]: Incorrect indentation.

File: Frends.Community.RabbitMQ/RabbitMQTask.cs [hunk 3]
```diff
@@ -216,16 +216,16 @@ public static Output ReadMessage([PropertyTab]ReadInputParams inputParams)
...
+            OutputString outString = new OutputString
+            {
+                Messages = messages.Messages.Select(m =>
+              new MessageString
+              {
+                  DeliveryTag = m.DeliveryTag,
+                  MessagesCount = m.MessagesCount,
+                      Data = Encoding.UTF8.GetString(Convert.FromBase64String(m.Data)),
+                      Headers = m.Headers
+                  }).ToList()
```
Comment (RikuVirtanen) [inline Frends.Community.RabbitMQ/RabbitMQTask.cs:228]: Incorrect indentation.

**Review by RikuVirtanen — CHANGES_REQUESTED** [review]
Some indentations where wrong but otherwise looking good. Please add some test to indicate that the bug is fixed.

**Comment by levik-buka** [comment]
I don't understand why, but I can not reproduce "basicProperties.IsHeadersPresent() returns true, but basicProperties.Headers is null" error any more.

**Pull request closed without merging**